packages/cxl-ui/scss/global/cxl-card.scss

```diff
@@ -0,0 +1,69 @@
+cxl-card {
+  box-sizing: border-box;
```

Comment:
Can we attempt to refactor without box model change, to start?

Reply:
Removing it works